modules/creating-manifest-file-multiple-interfaces-br-ex.adoc

```diff
@@ -0,0 +1,260 @@
++
+.Example configuration for the `extra_bridge` file, `/etc/ovnk/extra_bridge`, that references a secondary interface that is attached to a VLAN
+[source,text]
+----
+bond1
+----
+
+. Create a machine configuration manifest file that defines the existing static interface that hosts `br-ex1` on any nodes restarted on your cluster:
++
+.Example of a machine configuration file that defines `bond1` as the interface for hosting `br-ex1`
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 12-worker-extra-bridge
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+        - path: /etc/ovnk/extra_bridge
+          mode: 0420
+          overwrite: true
+          contents:
+            source: data:text/plain;charset=utf-8,bond1
+          filesystem: root
+----
+
+. Apply the machine-configuration to your selected nodes:
+
+[source,terminal]
+----
+oc create -f <machine_config_file_name>
+----
+
+. Optional: You can override the `br-ex` selection logic for nodes by creating a machine configuration file that in turn creates a `/var/lib/ovnk/iface_default_hint` resource. 
++
+[NOTE]
+====
+The resource lists the name of the interface that `br-ex` selects for your cluster. By default, `br-ex` selects the primary interface for a node based on boot order and the IP address subnet in the machine network. Certain machine network configurations might require that `br-ex` continues to select the default interfaces or bonds for a host node.
+====
++
+.. Create a machine configuration file on the host node to override the default interface.
++
+[IMPORTANT]
+====
+Only create this machine configuration file for the purposes of changing the `br-ex` selection logic. Using this file to change the IP addresses of existing nodes in your cluster is not supported.
+====
++
+.Example of a machine configuration file that overrides the default interface
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 12-worker-br-ex-override
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+        - path: /var/lib/ovnk/iface_default_hint
+          mode: 0420
+          overwrite: true
+          contents:
+            source: data:text/plain;charset=utf-8,bond0
+          filesystem: root
+----
++
+.. Reboot your host node.
++
+.. Before you apply the configuration to all new nodes, check that `br-ex` selects the intended interface and does not conflict with the new interfaces that you defined on `br-ex1`.
++
+.. Apply the machine configuration file to all new nodes in your cluster:
+
```

Comment:
add a plus to join this command inline. (+)

Reply:
:heavy_check_mark: